host/bcm283x/dma.go

```diff
@@ -743,13 +743,15 @@ func startPWMbyDMA(p *Pin, rng, data uint32) (*dmaChannel, *videocore.Mem, error
 	if drvDMA.dmaMemory == nil {
 		return nil, nil, errors.New("bcm283x-dma is not initialized; try running as root?")
 	}
-	cb, buf, err := allocateCB(2*32 + 4) // 2 CBs + mask
+
+	// Hardcoded len(controlBlock) == 32
+	cbBytes := uint32(32)
```

Comment:
can you make it a const? No idea why I hadn't done it before.

Reply:
Done